include/boost/url/grammar/sequence_rule.hpp

```diff
@@ -130,6 +131,98 @@ sequence_rule(
 }
 #endif
 
+#ifndef BOOST_URL_DOCS
+namespace detail {
+
+template<class Rule>
+struct squelch_rule_t
```

Comment:
It's an implementation detail anyway, is this term usual? I've never seen it in my life. I only undertood what it means after reading the code.